src/libs/Navigation/AppNavigator/AuthScreens.tsx

```diff
@@ -246,6 +235,10 @@ function AuthScreens({session, lastOpenedPublicRoomID, initialLastUpdateIDApplie
     // State to track whether the delegator's authentication is completed before displaying data
     const [isDelegatorFromOldDotIsReady, setIsDelegatorFromOldDotIsReady] = useState(false);
 
+    const [session] = useOnyx(ONYXKEYS.SESSION, {canBeMissing: true, allowStaleData: false});
```

Comment:
```suggestion
    const [session] = useOnyx(ONYXKEYS.SESSION, {canBeMissing: true});
```

We don't need to specify `allowStaleData` to `false`, since it's the default behavior.